src/core/util/env.js

```diff
@@ -8,12 +8,12 @@ export const inBrowser = typeof window !== 'undefined'
 export const inWeex = typeof WXEnvironment !== 'undefined' && !!WXEnvironment.platform
 export const weexPlatform = inWeex && WXEnvironment.platform.toLowerCase()
 export const UA = inBrowser && window.navigator.userAgent.toLowerCase()
-export const isIE = UA && /msie|trident/.test(UA)
-export const isIE9 = UA && UA.indexOf('msie 9.0') > 0
+export const isIE = inBrowser && document.documentMode
+export const isIE9 = inBrowser && document.documentMode && document.documentMode === 9
```

Comment:
`inBrowser && document.documentMode === 9` should be enough.

Reply:
fixed it, thank you!